ios/StatusPanel/Data/TFL/TFLDataSource.swift

```diff
@@ -103,6 +103,7 @@ final class TFLDataSource: DataSource {
         Line(id: "central", title: "Central Line", color: Color(hex: 0xe32017)),
         Line(id: "circle", title: "Circle Line", color: Color(hex: 0xffd300)),
         Line(id: "district", title: "District Line", color: Color(hex: 0x00782a)),
+        Line(id: "elizabeth", title: "Elizabeth Line", color: Color(hex: 0x9364cd)),
```

Comment:
So, https://content.tfl.gov.uk/elizabeth-line-design-idiom.pdf actually says this should be #6950a1. This disagrees with https://londonmymind.com/london-tube-colors/ (which is I guess where you got it from). I think we should go with the official TfL docs.

Reply:
nice catch!